src/elexmodel/cli.py

```diff
@@ -66,6 +66,13 @@
     help="options: results, data, config",
 )
 @click.option("--handle_unreporting", "handle_unreporting", default="drop", type=click.Choice(["drop", "zero"]))
+@click.option(
+    "--new_estimands",
```

Comment:
I would slightly prefer if we called this `estimand_fns` or something or `estimand_fn_dict`